OpenRA.Mods.Common/Activities/Air/HeliReturnToBase.cs

```diff
@@ -98,7 +98,7 @@ public override Activity Tick(Actor self)
 
 				return ActivityUtils.SequenceActivities(
 					new HeliFly(self, Target.FromPos(dest.CenterPosition + offset)),
-					new Turn(self, initialFacing),
+					aircraft.Info.TurnToLand ? new Turn(self, initialFacing) : null,
```

Comment:
`SequenceActivities` doesn't have any special logic for skipping `null`, so won't this break things?

Reply:
This `SequenceActivities` is replaced with a list later int he PR, so it would be better to remove this commit and split the check out from there into its own later commit.

Reply:
> SequenceActivities doesn't have any special logic for skipping null, so won't this break things?

Just for the record, I originally tested this without the landingProcedures list and it worked. I haven't bothered checking, but I assume our activity queue code automatically jumps to the next activity if the current one is null.